www/js/controllers/keypad_controller.js

```diff
@@ -0,0 +1,19 @@
+app.controller('KeyPadCtrl', function($scope, $state) {
+  $scope.amount = 0;
+
+  $scope.addHundred = function(key) {
+    $scope.amount = $scope.amount * 100;
```

Comment:
We need to have an upper limit on `amount`. Currently if you keep pressing buttons you eventually get `NaN` which is not good. Probably $99,999.99 is plenty! This will probably mean you'll need another method (should be private) called `setAmount` or something, which enforces this limit.